src/worldmap.test.ts

```diff
@@ -424,4 +424,33 @@ describe('Worldmap', () => {
       expect(worldMap.circles[2]._popup._content).toBe(`United States: ${HIGH}`);
     });
   });
+
+  describe('when variables have M or K', () => {
+    beforeEach(() => {
+      setupWorldmapFixture();
+    })
+    it('shouldnt break if only numbers are added', () => {
+      const input = '1000';
+      const output = worldMap.replaceThousandsAndMillions(input);
+      expect(output).toEqual(input);
+    });
+    it('should replace K with 1000', () => {
```

Comment:
Basically you have a parameterized unit test with 4 test case.
(_human readable name of the test case_, _input_, _expected output_)

[Here how I solved it in other project](https://github.com/HBOCodeLabs/Infra-Video-Job-Manager-Api/blob/086f6c497e3a616ac7bd51dae6ea95f0cd50476a/lambda/Video-Job-Manager-Api/test/unit-test/repository/index.spec.get-jobs-from-database.ts#L53)